src/components/PullRequests/PullRequest.js

```diff
@@ -29,7 +29,12 @@ export const PullRequest = ({ pullRequest }) => {
             <Labels labels={pullRequest.labels.nodes} />
           </div>
           <p className={styles.title}>
-            <a href={pullRequest.permalink}>{pullRequest.title}</a>
+            <a
+              href={pullRequest.permalink}
+              target="_blank"
```

Comment:
can you add `rel="noopener noreferrer"` to satisfy eslint please :)

Reply:
Done 👍